app/build.gradle

```diff
@@ -1,54 +1,40 @@
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
-
-def keystorePropertiesFile = rootProject.file("keystore.properties")
-def keystoreProperties = new Properties()
-
-keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-parcelize'
+    id 'kotlin-kapt'
+}
 
 android {
-    compileSdkVersion 30
-    buildToolsVersion "29.0.3"
+    namespace 'com.illusionware.npsbrowser'
+    compileSdkVersion 33
 
     defaultConfig {
         applicationId "com.illusionware.npsbrowser"
-        minSdkVersion 23
-        targetSdkVersion 30
+        minSdkVersion 24

```

Comment:
any reason for requiring 24 as the minimum sdk version?

Reply:
Vita3k minimum sdk is 24

Reply:
This is a standalone app and is not related to Vita3K in any way. 23 minimum is fine if there are no technical obstacles for it